src/libstd/primitive_docs.rs

```diff
@@ -1036,6 +1040,11 @@ mod prim_ref { }
 /// [`FnMut`]: ops/trait.FnMut.html
 /// [`FnOnce`]: ops/trait.FnOnce.html
 ///
+/// Function pointers are pointers that point to *code*, not data. They can be called
+/// just like functions. Like references, function pointers are assumed to not be null,
```

Comment:
and aligned?

Reply:
At least right now there is no such thing as an unaligned function pointer, and the nomicon also does not mention it. See https://github.com/rust-lang/unsafe-code-guidelines/issues/72.

If we said "aligned", we'd have to say aligned to which value. I think that's out-of-scope for this PR.

Reply:
Hmm... so should we use the same cop-out as for references?

Reply:
What wording do you propose? If we add anything here it should likely also be added to https://doc.rust-lang.org/nightly/nomicon/what-unsafe-does.html and https://doc.rust-lang.org/stable/reference/behavior-considered-undefined.html. But I don't think we can add anything useful, and at least currently we are consistent about not talking about fn pointer alignment... that's a can of worms I do not intend to open. ;)

Reply:
```suggestion
/// just like functions. Like references, function pointers are, among other things, assumed to not be null,
```
Weasel words 🎉

Reply:
I don't think this is very helpful, people will ask "what are those other things". I'm not a fan of "yoda-style" documentation (that opens more questions than it answers).

But whatever.